scaling_performance/install_practices.adoc

```diff
@@ -0,0 +1,79 @@
+[[scaling-performance-install-best-practices]]
+= Recommended Installation Practices
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-preinstalling-dependencies]]
+== Pre-installing Dependencies
+
+Save time by ensuring that the RPMs for `atomic-openshift-*`, `iptables`,
+`docker`, and other required packages have been set up before installing
+{product-title}. 
+
+If the dependencies have not been pre-installed, a node host will access the
+network to install the RPMs. Pre-installing dependencies creates a more
+efficient install by accessing the RPMS only when necessary, instead of a number
+of times per host during the install.
+
+This is also useful for machines that can't access the registry for security
+purposes.
+
+[[scaling-performance-install-optimization]]
+== Ansible Install Optimization
+
+{product-title} is installed using Ansible. Ansible is useful for running parallel operations, meaning a fast and efficient installation. However, these can be improved upon with additional tuning options.
+
+Parallel behavior can overwhelm a content source such as your image registry or
+Red Hat Satellite server. Preparing your servers infrastructure pods and
+operating system patches can help avoid this situation. See the
+xref:../install_config/install/advanced_install.html#configuring-ansible[Configuring
+Ansible section] for more information.
+
+Run the installer from the lowest-possible latency control node (LAN speeds).
+Running over a WAN is not advised, neither is running the install over a lossy
```

Comment:
@ahardin-rh That's what I thought at first as well, but it turns out "lossy" is a legit term that comes down to "a network with a bunch of stuff wrong with it". But thanks to your comment I asked around some of the network-smart docs people in the office and they said that lossy is established enough to have in the docs. If you have suggestions for something better, I'm all ears, though.